docs/api-testing/integrations/apifctl-cicd-integration.md

```diff
@@ -483,3 +483,6 @@ Specifies the number of events and metrics to be skipped from the beginning of t
 ```bash
 -o 10
 ```
+## More Information
+
+* [API Testing API Methods](/dev/api/api-testing/)
```

Comment:
```suggestion
* [API Testing Methods](/dev/api/api-testing/)
```

Reply:
@lysannep fyi "API Testing Methods" means nothing. The link is to "API Methods" for API Testing. It can be "API Methods" or "API Testing API Methods" but not "API Testing Methods"

Reply:
@lauramthomas @lysannep , Lory is right -- this needs to remain "API Testing API Methods" (as weird as it sounds, it's correct!)